composer.json

```diff
@@ -30,7 +30,7 @@
         "phpstan/phpstan-strict-rules": "^1.1",
         "doctrine/coding-standard": "^12",
         "doctrine/common": "^3.0",
-        "phpunit/phpunit": "^8.5 || ^9.5",
+        "phpunit/phpunit": "^8.5.38 || ^9.5",
```

Comment:
old versions of PHPUnit seem to struggle with `@requires` annotation that are included in a single-line comment.